.github/workflows/klocwork-pull-request.yml

```diff
@@ -102,17 +102,24 @@ jobs:
           if-no-files-found: error
       
       - name: get child issue ids
-        run: awk -F ';' '{print $1}' /home/klocwork/child_issues.txt |sort > /home/klocwork/child_issues_ids.txt
+        run: awk -F ';' '{print $1}' /home/klocwork/child_issues.txt | sort > /home/klocwork/child_issues_ids.txt
       
       - name: fixed issues
         run: |
           FIXED_ISSUE_ID=$(comm -13 /home/klocwork/child_issues_ids.txt /home/klocwork/parent_issues_ids.txt | sed -z 's/\n/,/g;s/,$/\n/')
-          kwcheck list --id ${FIXED_ISSUE_ID} -F detailed --project-dir /home/klocwork/.kwlp --settings-dir /home/klocwork/.kwps
+          echo kwcheck list --id ${FIXED_ISSUE_ID} -F detailed --project-dir /home/klocwork/.kwlp --settings-dir /home/klocwork/.kwps
+          if [ \"${FIXED_ISSUE_ID}\" != \"\" ]
```

Comment:
The escaping `\` is not needed. In bash ([which is the default](https://github.com/actions/runner/blob/main/docs/adrs/0277-run-action-shell-options.md)) you can use `-v` to check if a variable is set:
```
if [[ -v FIXED_ISSUE_ID ]]; then
```
(This has the advantage that it also works in `set -u` mode, though that is not the default in GitHub actions.)

Reply:
In this case I think the variable is set regardless, I have now used -n option for checking whether the given env var is empty.